presets/ras-defaults.json

```diff
@@ -105,7 +105,18 @@
                     "required": true,
                     "default": []
                 }
-            ]
+            ],
+            "help_info": {
+                "screenshot": "reg-overlay.png",
```

Comment:
I think it's best if this is left empty, if the image is not available it'll cause a 404. The prompt UI shouldn't render anything in that case.

Reply:
Ah, good call! d92742 removes the `screenshot` property if it can't find the referenced filename in the `src/assets` folder, which should avoid rendering the `img` tag and making a 404 request.